project-base/storefront/components/Forms/TextInput/SearchInput.tsx

```diff
@@ -35,7 +35,7 @@ export const SearchInput: FC<SearchInputProps> = ({
     };
 
     return (
-        <div className="relative w-full">
+        <div className="relative w-full rounded-md border border-borderAccent">
```

Comment:
please double-check, that these new classes don't affect other search inputs through the website (eg. `complaints.tsx` or `new-complaint.tsx`)

Reply:
Yes, this change will affect all pages where it's used. But it's updated based on our design where it's designed as components so I think it's ok to restyle it elsewhere too